indigo_app/views/works.py

```diff
@@ -629,9 +629,12 @@ def get_works(self, table, form):
                         work.save_with_revision(self.request.user)
 
                         # link publication document
+                        publication_number = work.publication_number
+                        if " " in work.publication_number:
```

Comment:
This feels a little dangerous to me, we'll have to see how other jurisdictions handle it.

Splitting on a string that doesn't have whitespace just returns the string, so you can safely do `publication_number = work.publication_number.split()[-1]` without having to check for whitespace.